### ISSUE TITLE
`alternativeText` and `caption` field for files are unnecessarily limited to 255 characters

### ISSUE THREAD

**Issue by phil-janeapp** [strapi/strapi#22385]
### Required System information

- Node.js version: v20.17.0
- NPM version: N/A
- Strapi version: 5.1.1
- Database: postgres
- Operating system: Linux
- Is your project Javascript or Typescript: Typescript

### Describe the bug

May be fixed in more recent version of v5, but the `alternativeText` and `caption` field for files are set to a 255 character limit on postgres (and possibly other datastores).  

Based on some quick research I did, it would appear that there's actually no browser/spec/screen reader limit for `alt` text for image, and that seems like the most-likely place for this value to be used.

### Steps to reproduce the behavior

1. Go to the media library or upload a file directly into a content-type
2. Edit the alternative text or caption to be longer than 255 characters
3. Submit the changes

### Expected behavior

That both the alternative text and caption fields support text longer than 255 characters

**Comment by spyderdsn** [comment]
I agree and faced the same problem. Please extend these fields by default.
